sdk/keyvault/azcertificates/constants.go

```diff
@@ -0,0 +1,113 @@
+	// DeletionRecoveryLevelCustomizedRecoverablePurgeable - Denotes a vault state in which deletion is recoverable, and which also permits immediate and permanent
+	// deletion (i.e. purge when 7<= SoftDeleteRetentionInDays < 90). This level guarantees the recoverability of the deleted entity during the retention interval,
+	// unless a Purge operation is requested, or the subscription is cancelled.
+	DeletionRecoveryLevelCustomizedRecoverablePurgeable DeletionRecoveryLevel = "CustomizedRecoverable+Purgeable"
+	// DeletionRecoveryLevelPurgeable - Denotes a vault state in which deletion is an irreversible operation, without the possibility for recovery. This level
+	// corresponds to no protection being available against a Delete operation; the data is irretrievably lost upon accepting a Delete operation at the entity
+	// level or higher (vault, resource group, subscription etc.)
+	DeletionRecoveryLevelPurgeable DeletionRecoveryLevel = "Purgeable"
+	// DeletionRecoveryLevelRecoverable - Denotes a vault state in which deletion is recoverable without the possibility for immediate and permanent deletion
+	// (i.e. purge). This level guarantees the recoverability of the deleted entity during the retention interval(90 days) and while the subscription is still
+	// available. System wil permanently delete it after 90 days, if not recovered
+	DeletionRecoveryLevelRecoverable DeletionRecoveryLevel = "Recoverable"
+	// DeletionRecoveryLevelRecoverableProtectedSubscription - Denotes a vault and subscription state in which deletion is recoverable within retention interval
+	// (90 days), immediate and permanent deletion (i.e. purge) is not permitted, and in which the subscription itself cannot be permanently canceled. System
+	// wil permanently delete it after 90 days, if not recovered
+	DeletionRecoveryLevelRecoverableProtectedSubscription DeletionRecoveryLevel = "Recoverable+ProtectedSubscription"
+	// DeletionRecoveryLevelRecoverablePurgeable - Denotes a vault state in which deletion is recoverable, and which also permits immediate and permanent deletion
+	// (i.e. purge). This level guarantees the recoverability of the deleted entity during the retention interval (90 days), unless a Purge operation is requested,
+	// or the subscription is cancelled. System wil permanently delete it after 90 days, if not recovered
+	DeletionRecoveryLevelRecoverablePurgeable DeletionRecoveryLevel = "Recoverable+Purgeable"
+)
+
+// ToPtr returns a *DeletionRecoveryLevel pointing to the current value.
+func (c DeletionRecoveryLevel) ToPtr() *DeletionRecoveryLevel {
+	return &c
+}
+
+// JSONWebKeyCurveName - Elliptic curve name. For valid values, see JsonWebKeyCurveName.
+type JSONWebKeyCurveName string
+
+const (
+	JSONWebKeyCurveNameP256  JSONWebKeyCurveName = "P-256"
+	JSONWebKeyCurveNameP256K JSONWebKeyCurveName = "P-256K"
+	JSONWebKeyCurveNameP384  JSONWebKeyCurveName = "P-384"
+	JSONWebKeyCurveNameP521  JSONWebKeyCurveName = "P-521"
+)
```

Comment:
`CertificateKeyCurveName`: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateKeyCurveName.cs

Basically the same as Keys' `KeyCurveName` but with `Certificate` prepended to avoid conflicts in most other languages that don't force a module reference like Go does.